modules/misc/xdg.nix

```diff
@@ -143,6 +143,7 @@ in {
         (mapAttrs' (name: file: nameValuePair "${cfg.stateHome}/${name}" file)
           cfg.stateFile)
         { "${cfg.cacheHome}/.keep".text = ""; }
+        { "${cfg.stateHome}/.keep".text = ""; }
```

Comment:
:-1: `stateHome` doesn't have a default value:
```
error: The option `home-manager.users.rschuetz.xdg.stateHome' was accessed but has no value defined. Try setting the option.
```

Reply:
@teto Since there seems to be no intention to implement the easy fix, can you please revert this PR?

Reply:
Never mind, here's the fix: https://github.com/nix-community/home-manager/pull/6555